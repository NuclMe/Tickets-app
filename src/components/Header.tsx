import Box from '@mui/material/Box';
import Link from '@mui/material/Link';
import type { FC } from 'react';

import airplane from '../assets/airplane.svg';
import { colors } from '../styledVars';

export const Header: FC = () => (
  <Box
    sx={{
      display: 'flex',
      justifyContent: 'center',
      mb: '50px',
    }}
  >
    <Link
      href="#"
      sx={{
        backgroundColor: `${colors.blue}`,
        borderRadius: '50%',
        padding: '15px',
        display: 'flex',
        alignItems: 'center',
        boxShadow: '0 2px 4px rgba(0, 0, 0, 0.8)',
      }}
    >
      <img src={airplane} alt="Airplane" style={{ width: 32, height: 32 }} />
    </Link>
  </Box>
);
