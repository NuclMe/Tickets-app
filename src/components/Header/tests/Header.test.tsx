import { render } from '@testing-library/react';

import { Header } from '../Header';

describe('Header', () => {
  it('renders the Header component via snapshot', () => {
    render(<Header />);

    expect(Header).toMatchSnapshot();
  });
});
