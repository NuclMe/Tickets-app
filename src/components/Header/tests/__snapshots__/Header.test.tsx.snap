// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > renders the Header component via snapshot 1`] = `[Function]`;
